server/helpers/cache.js

```diff
@@ -0,0 +1,38 @@
+const TIMEOUT = 24 * 60 * 60 * 1000 // 1 day
+const PRUNE_RATIO = 0.1
+
+class Cache {
+  constructor(limit, maxAge = TIMEOUT) {
+    this.limit = limit
+    this.maxAge = maxAge
+    this.cache = {}
+    this.size = 0
```

Comment:
why do we have to keep the size explicitly? Were you concerned about Object.keys().length being inefficient?

Reply:
Yes, from what I found, `Object.keys().length` is O(n)